modules/apps/server/server-admin-web-test/src/testIntegration/java/com/liferay/server/admin/web/internal/portlet/action/test/EditServerMVCActionCommandTest.java

```diff
@@ -421,6 +421,12 @@ public void testUpdateMail() {
 			_testUpdateMailPortletPreferences(
 				() -> Boolean.FALSE, () -> 0, () -> StringPool.BLANK,
 				portletPreferences);
+
+			_testUpdateMailWithAndWithoutSecureProtocol(
+				portletPreferences, Boolean.TRUE);
+
+			_testUpdateMailWithAndWithoutSecureProtocol(
+				portletPreferences, Boolean.FALSE);
```

Comment:
@jeffwu0724 Could you please change these `Boolean` usages to plain `true` or `false`? Including but not limited to the ones on line 422, 426, and here.

Line 422 expects a `BooleanSupplier`, which declares its method to return `boolean`; the method you added also expects a `boolean` not a `Boolean` as the parameter. Using `Boolean.TRUE/FALSE` introduces unnecessary auto-unboxing -- which doesn't hurt in a test, but really not needed.

Reply:
Thanks for the good advice! I have changed to use boolean instead of Boolean to avoid unnecessary auto-unboxing